src/main/java/io/getunleash/DefaultUnleash.java

```diff
@@ -179,56 +179,53 @@ private FeatureEvaluationResult getFeatureEvaluationResult(
-                    if (configuredStrategy == UNKNOWN_STRATEGY) {
-                        LOGGER.warn(
-                                "Unable to find matching strategy for toggle:{} strategy:{}",
-                                toggleName,
-                                strategy.getName());
-                    }
+            if (featureToggle.getStrategies().isEmpty()) {
+                return new FeatureEvaluationResult(
+                        true, VariantUtil.selectVariant(featureToggle, context, defaultVariant));
+            }
+            for (ActivationStrategy strategy : featureToggle.getStrategies()) {
+                Strategy configuredStrategy = getStrategy(strategy.getName());
+                if (configuredStrategy == UNKNOWN_STRATEGY) {
+                    LOGGER.warn(
+                            "Unable to find matching strategy for toggle:{} strategy:{}",
+                            toggleName,
+                            strategy.getName());
+                }
 
-                    FeatureEvaluationResult result =
-                            configuredStrategy.getResult(
-                                    strategy.getParameters(),
-                                    enhancedContext,
-                                    ConstraintMerger.mergeConstraints(featureRepository, strategy),
-                                    strategy.getVariants());
-
-                    if (result.isEnabled()) {
-                        Variant variant = result.getVariant();
-                        // If strategy variant is null, look for a variant in the featureToggle
-                        if (variant == null) {
-                            variant =
-                                    VariantUtil.selectVariant(
-                                            featureToggle, context, defaultVariant);
-                        }
-                        result.setVariant(variant);
-                        return result;
+                FeatureEvaluationResult result =
+                        configuredStrategy.getResult(
+                                strategy.getParameters(),
+                                enhancedContext,
+                                ConstraintMerger.mergeConstraints(featureRepository, strategy),
+                                strategy.getVariants());
+
+                if (result.isEnabled()) {
+                    Variant variant = result.getVariant();
+                    // If strategy variant is null, look for a variant in the featureToggle
+                    if (variant == null) {
+                        variant = VariantUtil.selectVariant(featureToggle, context, defaultVariant);
                     }
+                    result.setVariant(variant);
+                    return result;
                 }
             }
-            return new FeatureEvaluationResult(false, defaultVariant);
         }
+        return new FeatureEvaluationResult(false, defaultVariant);
```

Comment:
Something feels very subtly off here but it's not breaking tests sooo... it might be okay?

What's going through my head? Previously, we'd fall through to true, now we fall through to false. In theory, a toggle that has no strategies should fallback to the 'enabled' property, not true or false. That's distinct from a toggle that doesn't have strategies defined, which should never happen server side but when it does we turn off the toggle

So really we're into undefined territory here. It's probably okay but there's nothing in the client spec that enforces this behavior, so it's possible we break this again subtly